docs/maintainers/migrating-your-config-json-files.md

```diff
@@ -29,8 +29,21 @@ For v3, the existing `config.json` files will need to be updated. You can use th
 }
 ```
 
-4. Added concepts for all Practice Exercises
+As can be seen, the existing exercises are temporarily removed from the `config.json` file. They will return as Practice Exercises once the Concept Exercises have been added. More details will be added at a later stage.
 
-The existing exercises are temporarily removed from the `config.json` file. They will return as practice exercises once the concept exercises have been added.
+4. Remove the `"foregone"` property. The property will return once the Practice Exercises have been added.
 
-More details will be added at a later stage.
+## Conversion script
+
+All of the above changes, except for adding the online editor settings, can be applied automatically using the following [jq command][jq]:
+
+```
+jq 'del(.foregone)
+    | .exercises = {"concept": [], "practice": []}
```

Comment:
In that case, extend it to:
```suggestion
    | .exercises = {concept: [], practice: []}
    | .online_editor = {indent_style: "...", indent_size: N}
```

(I've used another feature of jq here, which is to omit quotes around keys when the key string allows it.)